tools/jil/bin/server.js

```diff
@@ -16,14 +16,10 @@ var driver = require('../index')
 
 var hostname = config.host
 driver.router.handle(driver.assetServer.defaultAgentConfig.licenseKey, true)
-
 driver.assetServer.renderIndex = true
-console.log('asset server: http://' + hostname + ':' + driver.assetServer.port)
-if (driver.assetServer.sslPort) {
-  console.log('asset server (SSL): https://' + hostname + ':' + driver.assetServer.sslPort)
-}
-console.log('fake router: http://' + hostname + ':' + driver.router.port)
-if (driver.router.sslPort) {
-  console.log('fake router (SSL): https://' + hostname + ':' + driver.router.sslPort)
-}
-console.log('secondary (cors) server: http://' + hostname + ':' + driver.assetServer.corsServer.port)
\ No newline at end of file
+
+setTimeout(() => {
+  console.log('asset server: http://' + hostname + ':' + driver.assetServer.port)
```

Comment:
Not often but because I believe it wasn't even functional/no easy way to enable it hence I never bothered trying to figure it out whenever those situation arise. It's more of a nice-to-have, but no big deal.